### PR TITLE
Fix Sonar Cloud Code smell for EasyInput

### LINKED ISSUE #182: Fix SonarCloud code smells for `EasyInput.jsx`
Code smells can be found here: https://sonarcloud.io/project/issues?directories=src%2Flib&fileUuids=AYsB72ZUWgZgXcKLGRLY&resolved=false&id=giorgosart_react-easy-edit

### PR DESCRIPTION
**Pull request by kareemmahlees**
closes #182

### CODE REVIEW

**Comment by kareemmahlees** [comment]
@giorgosart could you review?

**Review by giorgosart — CHANGES_REQUESTED** [review]
@kareemmahlees I think that you need to add `onfocus: PropTypes.func` in the propTypes section and leave the `onfocus: () => { }` in defaultProps

**Comment by kareemmahlees** [comment]
> @kareemmahlees I think that you need to add `onfocus: PropTypes.func` in the propTypes section and leave the `onfocus: () => { }` in defaultProps

but the code smell indicates that the error is in the default props!

**Comment by giorgosart** [comment]
@kareemmahlees [defaultProp "onfocus" has no corresponding propTypes declaration.](https://sonarcloud.io/project/issues?open=AYsB72jgWgZgXcKLGRMB&id=giorgosart_react-easy-edit) this states that a prop that has a default value for it is not declared in propsTypes.

**Comment by kareemmahlees** [comment]
@giorgosart done!

File: src/lib/EasyInput.jsx [hunk 1]
```diff
@@ -33,13 +33,14 @@ EasyInput.propTypes = {
   attributes: PropTypes.object,
   cssClassPrefix: PropTypes.string,
   onFocus: PropTypes.func,
-  onBlur: PropTypes.func
+  onBlur: PropTypes.func,
+  onfocus: PropTypes.func
```
Comment (giorgosart) [inline src/lib/EasyInput.jsx:37]: remove this line

File: src/lib/EasyInput.jsx [hunk 2]
```diff
@@ -33,13 +33,14 @@ EasyInput.propTypes = {
...
 };
 
 EasyInput.defaultProps = {
   attributes: {},
   placeholder: Globals.DEFAULT_PLACEHOLDER,
-  onfocus: () => { }
+  onfocus: ()=>{}
```
Comment (giorgosart) [inline src/lib/EasyInput.jsx:43]:
```suggestion
  onFocus: ()=>{}
```

**Review by giorgosart — COMMENTED** [review]
@kareemmahlees see my latest comments.

File: src/lib/EasyInput.jsx (hunk 1 above)
Comment (giorgosart) [inline src/lib/EasyInput.jsx:37]:
```suggestion
```

File: src/lib/EasyInput.jsx [hunk 3]
```diff
@@ -33,13 +33,13 @@ EasyInput.propTypes = {
   attributes: PropTypes.object,
   cssClassPrefix: PropTypes.string,
   onFocus: PropTypes.func,
-  onBlur: PropTypes.func
+  onBlur: PropTypes.func,
```
Comment (giorgosart) [inline src/lib/EasyInput.jsx:36]:
```suggestion
  onBlur: PropTypes.func
```

**Review by giorgosart — APPROVED** [review]

**Pull request merged**